external/source/shellcode/windows/x86/src/crc32_hash.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python2.7

```

Comment:
This looks good, but it will need to be compatible with Python 3 since it is a local tool.